config/config.yaml

```diff
@@ -3,6 +3,8 @@ db:
   synchronize: false
 app:
   port: <port>
+  schedulerInterval: <cron interval>
```

Comment:
We should not make this configurable. I don't see any other indexer or similar service exposing internal configs like these. My philosophy is to be conservative, only expose what is required, and not bombard users with too many config options.

On the other hand, I agree that a 10s scheduling is too long so please change it to 1 second.

Reply:
If you are fine with it, i could make it `optional` which would also  align with your philosophy of being conservative. 

Also i think 10 seconds is quite fine :smile: